### DISCUSSION TITLE
display a website inside Ammo-world

### DISCUSSION THREAD

**Discussion by akerb00m** [cables-gl/cables_docs#504] [Answers & Questions] [answered]
Greetings !
Would it be possible to display a (functioning) website within the cables ammo world?
For instance, i would like to build a gallery in cables.gl to display the art i collected on FXhash, so a fxhash artworks would be displayed on a wall and it would be possible to 'run' them. 
i have tried to achieve this with the 'i frame' opp but it doesn't seem possible, maybe there is another way ?
cheers !
GJ

**Comment by steam0r** [comment]
we are doing something similar to what i think you want in this patch, maybe check it:

https://cables.gl/p/FmFMz4

**Reply by akerb00m** [reply]
Oh i definitely saw that, its amazing ! 
My next project will be an Ammo world (with sub patches) 

but for another project, i was hoping it would somehow be possible to create an (empty) gallery where people could load (part of) their fxhash collection into pre-made sloths and have them 'play'  I don't think something like that exists yet, and there might be some interest for it. 

Cheers !

**Comment by pandrr** [comment, marked as answer]
it is not possible to convert a website into texture directly

**Reply by akerb00m** [reply]
aight, thank you, then i wil stop trying : )

**Reply by kdklv** [reply]
what about rendering dom to canvas? i think i saw few examples, but im skeptical about compatibility 

http://html2canvas.hertzen.com/
https://codepen.io/srapp/pen/MyrpPe

**Reply by pandrr** [reply]
as i said not "directly"

also iframe content or a different website is not in the dom
